app/controllers/api/v2/access_requests_controller.rb

```diff
@@ -25,11 +25,13 @@ def index
       end
 
       def create
-        authorize AccessRequest # todo, generalise admin auth
         access_request = AccessRequest.new(access_request_params)
-        access_request.update(requester: @current_user,
-                              request_date_utc: Time.now.utc,
-                              status: :requested)
+        authorize access_request
+
+        access_request.requester        = User.find_by(email: access_request.requester_email)
+        access_request.request_date_utc = Time.now.utc
+        access_request.status           = :requested
+        access_request.save!
```

Comment:
Does it meant to do the bang?

Reply:
yeah. it should blow up since we dont handle validation errors